src/main/java/org/jclouds/vcloud/director/v1_5/compute/strategy/VCloudDirectorComputeServiceAdapter.java

```diff
@@ -438,4 +427,26 @@ public void suspendNode(String id) {
       api.getVmApi().suspend(id);
    }
 
+   private Vdc getVdc(String locationId) {
+      return api.getVdcApi().get(URI.create(locationId));
+   }
+
+   private Set<Vm> getAvailableVMsFromVAppTemplate(VAppTemplate vAppTemplate) {
+      return ImmutableSet.copyOf(filter(vAppTemplate.getChildren(), new Predicate<Vm>() {
+         // filter out vms in the vApp template with computer name that contains underscores, dots,
```

Comment:
Why do we do filter them out?
(I see the code was already there, just minor changes so this won't block merging).